src/SIL.Machine.AspNetCore/Services/ClearMLNmtEngineBuildJob.cs

```diff
@@ -258,7 +258,6 @@ async IAsyncEnumerable<Pretranslation> ProcessRowsAsync()
                     if (
                         (corpus.PretranslateAll || corpus.PretranslateTextIds.Contains(row.TextId))
                         && row.SourceSegment.Count > 0
-                        && row.TargetSegment.Count == 0
```

Comment:
This check should not be removed. We purposefully don't generate pretranslations for verses that already have a translation.

Reply:
What is the rationale behind that?  I can think of a few examples when people might want to:
* To manually compare against what is already written
* If it was drafted but we wanted to see what a new draft would look like with a new model

Basically, this behavior could be replicated through just not uploading the specific texts for training - yet since we are specifically calling out which texts to pretranslate, it makes sense to me to translate all of it.  By contrast, we could also add another flag called "forcePretranslate/pretranslateExistingText" or some other term that would have a description.  Whatever we decide now, we should at least update the documentation (or track the updating of it).

Reply:
Revert it for now - as per Damien "It will generate about the same text already".  Update Swagger documentation to specify this behavior.